autofs-5.1.0-beta1 - fix ldap default master map name config

From: Ian Kent <ikent@redhat.com>

If there is no master map name defined in the autofs configuration the
lookup will try to find a map with the default name and if the map isn't
found will substute dots with underscores and repeat the lookup.

So don't set the default master map name at defaults initialization time.
---
 CHANGELOG      |    1 +
 lib/defaults.c |    5 -----
 2 files changed, 1 insertion(+), 5 deletions(-)

--- autofs-5.0.5.orig/CHANGELOG
+++ autofs-5.0.5/CHANGELOG
@@ -205,6 +205,7 @@
 - fix wildcard key lookup.
 - fix out of order amd timestamp lookup.
 - fix ldap default schema config.
+- fix ldap default master map name config.
 
 03/09/2009 autofs-5.0.5
 -----------------------
--- autofs-5.0.5.orig/lib/defaults.c
+++ autofs-5.0.5/lib/defaults.c
@@ -266,11 +266,6 @@ static int conf_load_autofs_defaults(voi
 	const char *sec = autofs_gbl_sec;
 	int ret;
 
-	ret = conf_update(sec, NAME_MASTER_MAP,
-			  DEFAULT_MASTER_MAP_NAME, CONF_ENV);
-	if (ret == CFG_FAIL)
-		goto error;
-
 	ret = conf_update(sec, NAME_TIMEOUT,
 			  DEFAULT_TIMEOUT, CONF_ENV);
 	if (ret == CFG_FAIL)
